e2e/session.test.ts

```diff
@@ -0,0 +1,282 @@
+    .locator(
+      '.ant-form-item-control-input-content > .ant-select > .ant-select-selector',
+    )
+    .locator('input');
+  await expect(ResourcePreset).toBeVisible();
+  await ResourcePreset.fill('minimum');
+  await page.locator('.ant-select-dropdown:has-text("minimum")').click();
+  // launch
+  await page.getByRole('button', { name: 'Skip to review' }).click();
+
+  await page.locator('button').filter({ hasText: 'Launch' }).click();
+  await expect(page.locator('.ant-modal-confirm-title')).toHaveText(
+    'No storage folder is mounted',
+  );
+  await page.getByRole('button', { name: 'Start' }).click();
+};
+
+const createBatchSessionOnSessionStartPage = async (
+  page: Page,
+  sessionName: string,
+) => {
+  const batchRadioButton = page
+    .locator('label')
+    .filter({ hasText: 'Batch' })
+    .locator('input[type="radio"]');
+  await batchRadioButton.check();
+  const sessionNameInput = page.locator('#sessionName');
+  await sessionNameInput.fill(sessionName);
+  const BatchModeConfigurationCard = page.locator(
+    'div.ant-card:has-text("Batch Mode Configuration")',
+  );
+  expect(BatchModeConfigurationCard).toBeVisible();
+  await BatchModeConfigurationCard.getByLabel('Startup Command').fill(
+    'sleep 30',
+  );
+  await page.getByRole('button', { name: 'Next' }).click();
+  // select Minimum Requirements
+  const ResourcePreset = page
+    .locator('.ant-form-item-row:has-text("Resource Presets")')
+    .locator(
+      '.ant-form-item-control-input-content > .ant-select > .ant-select-selector',
+    )
+    .locator('input');
+  await expect(ResourcePreset).toBeVisible();
+  await ResourcePreset.fill('minimum');
+  await page.locator('.ant-select-dropdown:has-text("minimum")').click();
+  // launch
+  await page.getByRole('button', { name: 'Skip to review' }).click();
+  await page.locator('button').filter({ hasText: 'Launch' }).click();
+  await expect(page.locator('.ant-modal-confirm-title')).toHaveText(
+    'No storage folder is mounted',
+  );
+  await page.getByRole('button', { name: 'Start' }).click();
+};
+
+test.describe('Session Creation', () => {
+  const sessionName = 'e2e-test-session' + new Date().getTime();
+
+  test.beforeEach(async ({ page }) => {
+    await loginAsUser(page);
+  });
+  test.afterEach(async ({ page }) => {
+    // delete session after each test
+    await deleteSession(page, sessionName);
+  });
+
+  test('User can create interactive session on the Start page', async ({
+    page,
+  }) => {
+    // move to start page
+    await getMenuItem(page, 'start').click();
+    await expect(page).toHaveURL(/\/start/);
+    // click Start Session button and move to session start page
+    const card = getCardItemByCardTitle(page, 'Start Interactive Session');
+    expect(card).toBeVisible();
+    const creationButton = card.getByRole('button', {
+      name: 'Start Session',
+    });
+    await creationButton.click();
+    await expect(page).toHaveURL(/\/session\/start/);
```

Comment:
We need to wait until the network status is idle. This is because the request to load the resource group is sent after rendering.
```suggestion
    await page.waitForLoadState('networkidle');
```